from robo_ai.exception.api_error import ApiError


class InvalidTokenError(ApiError):
    pass
